Modules/TOF/src/CheckRawTime.cxx

```diff
@@ -57,7 +57,8 @@ Quality CheckRawTime::check(std::map<std::string, std::shared_ptr<MonitorObject>
       const Int_t highBinId = h->GetXaxis()->FindBin(mMaxRawTime);
       mRawTimePeakIntegral = h->Integral(lowBinId, highBinId);
       mRawTimeIntegral = h->Integral(1, h->GetNbinsX());
-      if ((mRawTimeMean > mMinRawTime) && (mRawTimeMean < mMaxRawTime)) {
+      result = Quality::Good;
```

Comment:
This can be bypassed by updating the limits, i.e. putting default values for the check that would be always true, then we can go back to it by changing the limits alone